src/common/config_parser.cpp

```diff
@@ -629,8 +629,12 @@ void ConfigParser::addOptionsTranslation(cli::CLIWrapper& cli) {
   addSuboptionsDevices(cli);
   addSuboptionsBatching(cli);
 
-  cli.add<bool>("--optimize",
-      "Optimize speed aggressively sacrificing memory or precision");
+  cli.add<bool>("--optimize16",
+      "Optimize speed aggressively sacrificing memory or precision by using 16bit integer GEMM with intgemm instead of floats. Only available on CPU");
```

Comment:
If it's only available on CPU, shall we add these options only to a build with CPU support?

Reply:
I think these should exist in GPU builds with an error message if used, in preparation for 8-bit int tensor core support on Turing and newer arches.

Reply:
Good point. We already have `--fp16`, which corresponds to `--precision float16`. Maybe `--optimize8/16` could/should be combined with the `--precision` option? @emjotde, what is the convention?

Reply:
I personally like the --precision integer8/integer16/integer8-shifted, but will wait for @emjotde point.